docs/content/how-to/azure/create-azure-cluster-with-options.md

```diff
@@ -100,3 +100,38 @@ spec:
 status:
   replicas: 0
 ```
+
+## Enabling Ephemeral OS Disks on Azure VMs
+To enable the ephemeral OS disk option on the Azure VMs in your HostedCluster, set the `enable-ephemeral-disk` flag to true.
+
+!!! important
+
+    Ephermeral OS disks are not available in every region or for every instance type. You may also need to adjust 
+    the disk size accordingly; to adjust the disk size use the `disk-storage-account-type` flag as shown in the example below.
```

Comment:
Just curious, how does the disk storage account type adjust the disk size?

Reply:
It doesn't. I typo-ed that and meant disk storage account type.